svm_controller.py

```diff
@@ -0,0 +1,31 @@
+from flask import Flask, request
+import pickle
+import numpy as np
+
+app = Flask(__name__)
+
+model = pickle.load(open("svv.sav", 'rb'))
+
+@app.route("/digits", methods = ["GET"])
+def get_app_description():
+    return "This is a digits recogniton app.", 200
+
+@app.route("/digits", methods = ["POST"])
+def get_digit():
+    request_data = request.get_json()
+    digit = request_data.get("digit")
+    if not digit or len(digit) != 64 or type(digit) != list: return "Incorrect json body", 400
+
+    for element in digit:
+        if not isinstance(element, int) or not all(ch in ['0', '1'] for ch in bin(element)[2:]):
```

Comment:
`all(ch in ['0', '1'] for ch in bin(element)[2:])` will always be true for integers

Reply:
Fixed